### DISCUSSION TITLE
Inflight vulnerability

### DISCUSSION THREAD

**Discussion by satinder1624** [babel/babel#16444] [General]
Hi,

I am getting a security concern warning from @babel/cli@7.24.1 > glob@7.2.3 > inflight@1.0.6

What I can do?

Thanks,

**Comment by nicolo-ribaudo** [comment]
The vulnerability only happens when `inflight` is directly called by an attacker. The way Babel uses Glob, and Glob uses inflight is not affected by the vulnerability, and you can safely ignore it.

**Comment by satinder1624** [comment]
We are using a vulnerability portal and there it shows that as a
vulnerability and my task is to take out that vulnerability. So any
guidance much appreciate it

**Reply by nicolo-ribaudo** [reply]
The vulnerability doesn't affect you though -- you should mark it as such in the portal.

**Comment by satinder1624** [comment]
I cannot mark it, that’s the problem. If it is exists it shows on the portal.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by nicolo-ribaudo** [reply]
What portal are you using? If it's Sync, you can configure it to mark a vulnerability as not applicable: https://docs.snyk.io/manage-risk/prioritize-your-issues/ignore-issues#decisions-to-ignore-issues, https://docs.snyk.io/manage-risk/prioritize-your-issues/ignore-issues#use-the-.snyk-file-to-ignore-issues
